packages/components/src/mobile-promo-card/index.tsx

```diff
@@ -64,46 +133,61 @@ export default function MobilePromoCard( { className, isWoo }: MobilePromoCardPr
 	const getPromoImage = () => {
 		if ( isApple ) {
 			const appStoreLink = isWoo
-				? 'https://apps.apple.com/ca/app/woocommerce/id1389130815'
-				: 'https://apps.apple.com/ca/app/jetpack-website-builder/id1565481562';
+				? getRedirectUrlPrivate( URL_WOO_APPLE )
+				: getRedirectUrlPrivate( URL_JETPACK_APPLE );
+			const tracksEventName = isWoo ? EVENT_WOO_CLICK_APPLE : EVENT_JETPACK_CLICK_APPLE;
 			return (
-				<a href={ appStoreLink }>
+				<a href={ appStoreLink } onClick={ () => recordTracksEventPrivate( tracksEventName ) }>
 					<img
 						className="promo-store-badge"
 						src={ storeBadgeApple }
 						alt="Badge for the Apple App Store"
 					/>
 				</a>
 			);
-		} else if ( isGoogle ) {
+		}
+		if ( isGoogle ) {
 			const appStoreLink = isWoo
-				? 'https://play.google.com/store/apps/details?id=com.woocommerce.android'
-				: 'https://play.google.com/store/apps/details?id=com.jetpack.android';
+				? getRedirectUrlPrivate( URL_WOO_GOOGLE )
+				: getRedirectUrlPrivate( URL_JETPACK_GOOGLE );
+			const tracksEventName = isWoo ? EVENT_WOO_CLICK_GOOGLE : EVENT_JETPACK_CLICK_GOOGLE;
 			return (
-				<a href={ appStoreLink }>
+				<a href={ appStoreLink } onClick={ () => recordTracksEventPrivate( tracksEventName ) }>
 					<img
 						className="promo-store-badge"
 						src={ storeBadgeGoogle }
 						alt="Badge for the Google Play Store"
 					/>
 				</a>
 			);
-		} else if ( isWoo ) {
+		}
+		if ( isWoo ) {
 			return <img className="promo-qr-code" src={ qrCodeWoo } alt="QR Code for Woo mobile app" />;
 		}
 		return (
 			<img className="promo-qr-code" src={ qrCodeJetpack } alt="QR Code for Jetpack mobile app" />
 		);
 	};
 
+	// Track "views" of this card.
+	if ( isWoo ) {
+		recordTracksEventPrivate( EVENT_WOO_VIEW );
+	} else {
+		recordTracksEventPrivate( EVENT_JETPACK_VIEW );
+	}
```

Comment:
We'll want these events to fire only upon mounting using [useEffect](https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect), like so:

```suggestion
	// Track "views" of this card.
	useEffect( () => {
		if ( isWoo ) {
			recordTracksEventPrivate( EVENT_WOO_VIEW );
		} else {
			recordTracksEventPrivate( EVENT_JETPACK_VIEW );
		}
	}, [ isWoo ] );
```